app/code/Magento/Contact/Model/Contact.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Magento\Contact\Model;
+
+use Magento\Contact\Api\ContactInterface;
+use Magento\Contact\Api\Data\ContactFormInterface;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * Class Contact
+ * @package Phoenix\Contact\Model
+ */
+class Contact implements ContactInterface
+{
+    /**
+     * @var MailInterface
+     */
+    private $mail;
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * Contact constructor.
+     * @param MailInterface $mail
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(
+        MailInterface $mail,
+        SerializerInterface $serializer
+    ) {
+        $this->mail = $mail;
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function send(ContactFormInterface $data)
+    {
+        $result = [];
+        try {
+            $this->mail->send($data->getEmail(), ['data' => new DataObject($data->__toArray())]);
```

Comment:
Please don't use NOT contract methods `__toArray`. If somebody provides own implementation of `ContactInterface`, this code will be broken.
We should depend on abstraction but not on the concrete implementation.

Look at `\Magento\Framework\Reflection\DataObjectProcessor::buildOutputDataArray` (or data can be extracted mannualy via getters if mailer don't have extension points in data)